Podcast/FeedViewController.swift

```diff
@@ -58,11 +58,14 @@ class FeedViewController: ViewController {
         feedTableView.refreshControl?.addTarget(self, action: #selector(fetchCards), for: .valueChanged)
         
         fetchCards()
+
+        navigationController?.heroNavigationAnimationType = .selectBy(presenting: .zoom, dismissing: .zoomOut)
```

Comment:
![](https://media.giphy.com/media/l0DAIDd1sdGhA38R2/giphy.gif)
i'm getting this weird glitch on returning to feed from series detail

Reply:
Will fix by not refreshing the feed every time we look at it